guides/V1_get_my_space_templates.md

```diff
@@ -0,0 +1,38 @@
+---
+copyright: 'Copyright IBM Corp. 2018'
+link: 'my-space-templates'
+is: 'experimental'
+---
+
+# Get My Space Templates
+
+Using the _mySpaceTemplates_ query, a user can fetch a collection of the custom space templates that they have created.
```

Comment:
>the custom space templates

Do we need the word "custom" here?  Not sure we have some that are custom and some that are not and distinguish them in queries...

Reply:
Removed.